feature_engine/mathematical_combination.py

```diff
@@ -0,0 +1,94 @@
+from feature_engine.base_transformers import BaseNumericalTransformer
+
+
+class MathematicalVariableCombinator(BaseNumericalTransformer):
+    """
+    The MathematicalVariableCombinator() applies basic mathematical operations across features,
+    returning one additional feature as a result.
```

Comment:
returning 1 or more additional features as a result. For example, if we have the variables number_payments_first_quarter, number_payments_second_quarter, number_payments_third_quarter and number_payments_fourth_quarter, we can use the MathematicalVariableCombinator to calculate the total number of payments and mean number of payments as follows:

transformer = MathematicalCombinator(variables=['number_payments_first_quarter', 'number_payments_second_quarte'r, 'number_payments_third_quarter', 'number_payments_fourth_quarter'], math_operations=['sum','mean'], new_variables_name=['total_number_payments', 'mean_number_payments'])

trasnformer.fit_transform(X)

The transformed X will contain the additional features total_number_payments and mean_number_payments, plus the original set of variables.

Could you add all of the above to the docstrings?

Reply:
yes, thanks